crates/api_models/src/analytics/search.rs

```diff
@@ -0,0 +1,4 @@
+#[derive(Clone, Debug, Default, serde::Deserialize, serde::Serialize)]
+pub struct SearchFilters {
+    pub payment_method: Vec<String>,
```

Comment:
```suggestion
    pub payment_method: Option<Vec<String>>,
```

let's make this option,

`payment_method in []` would be represented by Some(vec[]) whereas no filter would be represented by None